### DISCUSSION TITLE
node power off instead of deletion (Slurm)

### DISCUSSION THREAD

**Discussion by javierbq** [GoogleCloudPlatform/cluster-toolkit#2967] [Ideas and feature requests]
is it possible to configure a dynamic partition to power off idle nodes instead of deleting them? I have jobs where there is a big up front cost of downloading a database to an ssd, it would be great if can reuse those nodes.

**Comment by rohitramu** [comment]
Unfortunately, this is not possible right now.  I have internally filed a feature request so we can evaluate possible solutions and implementation details.

**Reply by javierbq** [reply]
thanks!
